docs/source/1-getting_started/Support.rst

```diff
@@ -0,0 +1,27 @@
+User Support
+===
+
+Do you:-
+* have a problem that you're having trouble solving? 
+* have some software you need installed? 
+* feel unsure about the best or preferred way to launch a job or use OzSTAR? 
+* have some code that is crashing or not working as expected (But has worked for you before or on other systems)?
+* are experiencing something mysterious that you don't fully understand?
+
+Feel free to send a ticket through to our user support email address: hpc-support@swin.edu.au
+
+We do not write code for you, and we only provide best effort support.
```

Comment:
Our goal is to provide best effort support and to get back to you as quickly as possible. However, some requests will be out of scope, e.g. don't ask us to rewrite your code for you, and will be politely declined.